src/Sidekick.Domain/Game/StatTranslations/AlternateStatProvider.cs

```diff
@@ -0,0 +1,77 @@
+using System;
```

Comment:
The implementation of this file should be in the Infrastructure project. You can leave the interface in the Domain project.